sdk/go.mod

```diff
@@ -11,3 +11,18 @@ require (
 	github.com/stretchr/testify v1.7.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+require (
```

Comment:
❓ are these supposed to be added here?

Reply:
Go 1.17 changed the layout of the file a bit. I don't remember the details.